TenancyInformationApi/V1/UseCase/GetTenancyByIdUseCase.cs

```diff
@@ -0,0 +1,22 @@
+using TenancyInformationApi.V1.Boundary.Response;
+using TenancyInformationApi.V1.Factories;
+using TenancyInformationApi.V1.Gateways;
+using TenancyInformationApi.V1.UseCase.Interfaces;
+
+namespace TenancyInformationApi.V1.UseCase
+{
+    public class GetTenancyByIdUseCase : IGetTenancyByIdUseCase
+    {
+        private readonly ITenancyGateway _gateway;
+        public GetTenancyByIdUseCase(ITenancyGateway gateway)
+        {
+            _gateway = gateway;
+        }
+
+        public TenancyInformationResponse Execute(string id)
+        {
+            var result = _gateway.GetById(id).ToResponse();
```

Comment:
Could the result of `_gateway.GetById(id)` be null and hence throw an error when calling `.ToResponse()`?

Reply:
The assignments in `ToResponse()` all use null-conditional operators, so if there is no `Tenancy` returned by the Gateway, it will return a new `TenancyInformationResponse` full of `null` members, 

I've written a new test to make sure the Use Case still returns the object.